deploy/kubernetes/overlays/kind/kustomization.yaml

```diff
@@ -0,0 +1,57 @@
+# Use the base config files as our foundation
+resources:
+  - ../../base
+
+# Deletes volume mounts from the plugin.yaml base config DaemonSet.
+# The Kind environment does not have volume access to the /dev, /mnt, /usr/bin, /usr/sbin, /usr/lib, /usr/lib64
+# directories on the host that are used by the lustre client.
+patchesStrategicMerge:
+  - |-
+    apiVersion: apps/v1
+    kind: DaemonSet
+    metadata:
+      name: lustre-csi-node
+    spec:
+      template:
+        spec:
+          volumes:
+            - name: dev-dir
```

Comment:
Take out /sbin and /bin as well.

Reply:
And what's happening with plugins-dir  and mountpoint-dir?  I see them in the 'volumes' list but they don't appear in any volumeMounts list.

Reply:
> Take out /sbin and /bin as well.

Done.

> And what's happening with plugins-dir and mountpoint-dir?

I deleted them via `$patch` here too. I'm not sure if those directories exist in a Kind environment, but I don't think they're needed anyways for a mock run.

> I see them in the 'volumes' list but they don't appear in any volumeMounts list.

What do you mean? In plugin.yaml, they are in both the volumes and volumeMounts of the csi-node-driver container.

Reply:
Check the output of:
`bin/kustomize build deploy/kubernetes/overlays/kind | less`